packages/discord.js/src/structures/Message.js

```diff
@@ -611,6 +611,24 @@ class Message extends Base {
     );
   }
 
+  /**
+   * Whether the message is bulk deletable by the client user
+   * @type {boolean}
+   * @readonly
+   * @example
+   * // Filter for bulk deletable messages
+   * channel.bulkDelete(messages.filter(message => message.bulkDeletable));
+   */
+  get bulkDeletable() {
+    const permissions = this.channel?.permissionsFor(this.client.user);
+    return (
+      this.deletable &&
+      this.inGuild() &&
+      permissions.has(PermissionFlagsBits.ManageMessages) &&
```

Comment:
Due to the optional chaining above, `permissions` is nullable.

Reply:
It's nullable if the message from a DM. In guild it always will be there since the guild member of the client is cached.